questions/src/main/java/com/ds/questions/FindThreeLargestNumber.java

```diff
@@ -16,8 +16,19 @@ public static void main(String[] args) {
     }
 
     private static int[] findThreeLargestNumber(int[] inputData) {
+        if (inputData == null || inputData.length == 0) {
+            return new int[]{0, 0};
+        } else {
+            for (int i = 0; i < 3; i++) {
+                for (int j = i + 1; j <= inputData.length - 1; j++)
+                    if (inputData[j] > inputData[i]) {
+                        int temp = inputData[j];
+                        inputData[j] = inputData[i];
+                        inputData[i] = temp;
+                    }
+            }
+        }
+        return inputData;
```

Comment:
Output should be only largest 3 numbers out of n numbers in `inputData`

Reply:
Yes, In response I will return new array of three sizes only which has the three largest numbers from an array.